src/pages/RunDetails.js

```diff
@@ -90,31 +104,40 @@ export const RunDetails = ({ submissionId, workflowId }) => {
   const fetchMetadata = useCallback(workflowId => Ajax(signal).Cromwell.workflows(workflowId).metadata({ includeKey, excludeKey }), [includeKey, excludeKey, signal])
 
   const loadWorkflow = useCallback(async (workflowId, updateWorkflowPath = undefined) => {
+    let failedTasks = {}
     const metadata = await fetchMetadata(workflowId)
+    if (metadata?.status?.toLocaleLowerCase() === 'failed') {
```

Comment:
Thinking out loud: Would be nice to have typescript enums here so we don't have to map the string statuses into actual cromwell states in more than one place. We should keep an eye out on how to combine this with the stuff in `job-common.js` when we move RunDetails over.